src/richie/plugins/lti_consumer/templates/richie/lti_consumer/lti_consumer.html

```diff
@@ -1,5 +1,5 @@
 <div
   class="richie-react richie-react--lti-consumer aspect-ratio"
-  style="{{ instance.inline_ratio_css_padding_bottom }}"
+  style="padding-bottom: {% widthratio instance.get_inline_ratio 1 100 %}%"
```

Comment:
There is a tag for that! I see that the value is rounded to the nearest integer. It does make sens to interpolate values in pixels but for percentage values, it bothers me a bit.
What do you think to simply multiply by 100 the `inline_ratio` value?

Reply:
It's the `widthratio` tag that rounds it down :thinking: If you think the precision is important, I will compute it myself on the model.

Reply:
I prefer to have our own computation and preserver precision.